.github/workflows/build-ws-container.yml

```diff
@@ -6,14 +6,25 @@ on:
   # can be run manually on https://github.com/cockpit-project/cockpit/actions
   workflow_dispatch:
 
+env:
+  tag: ${{ vars.REGISTRY_REPO || 'quay.io/cockpit/ws' }}
+
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        build:
+          - label: amd64
+            runner: ubuntu-24.04
+          - label: arm64
+            runner: ubuntu-24.04-arm
```

Comment:
Not too long ago I struggled with moving all our hardcoded versions to -latest, as the former tend to get out of date. E.g. ubuntu-20.04 was dropped recently. It really pains me to see that reintroduced here, but alas, 
[ubuntu-latest-arm is not a thing](https://github.com/orgs/community/discussions/148648#discussioncomment-11858187). So *sigh* I suppose we need to live with this.